### PR TITLE
canto-lending: remove LPs from tvl

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/canto-lending exports TVL: 
        
 
 
    canto                     21.47 M
    canto-borrowed            763.91 k
    borrowed                  763.91 k
    
    total                    21.47 M

**Comment by llamatester** [comment]
The adapter at projects/smartcredit.js exports TVL: 
        
 
 
    ethereum-borrowed         0
    ethereum                  731.11 k
    borrowed                  0
    
    total                    731.11 k

**Comment by llamatester** [comment]
The adapter at projects/canto-lending exports TVL: 
        
 
 
    canto-borrowed            763.91 k
    canto                     21.47 M
    borrowed                  763.91 k
    
    total                    21.47 M

**Comment by llamatester** [comment]
The adapter at projects/smartcredit.js exports TVL: 
        
 
 
    ethereum-borrowed         266.49 k
    ethereum                  731.11 k
    borrowed                  266.49 k
    
    total                    731.11 k

File: projects/smartcredit.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ module.exports = {
   timetravel: false,
   ethereum: {
     tvl,
-    staking,
+    // staking,
```
Comment (0xngmi) [inline projects/smartcredit.js:60]: why remove staking too?

**Pull request merged**